tests/unit/_experimental/test_plots.py

```diff
@@ -546,7 +2127,9 @@ def test__confusion__matrix(
 )
 def test__confusion__matrix__fails(
     test_name: str,
-    matchings: MulticlassInferenceMatches,
+    matchings: List[MulticlassInferenceMatches],
 ) -> None:
+    from kolena._experimental.object_detection.utils import compute_confusion_matrix_plot
```

Comment:
It seems like this import is moved within the scope of each confusion matrix test functions.  What's the reason behind this change?

Reply:
`kolena/_experimental/object_detection/utils.py` imports sklearn, so if the import is outside of the pytest metrics markers, then it will fail with import errors